helix-core/src/main/java/org/apache/helix/manager/zk/ZkSessionMismatchedException.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.helix.manager.zk;
+
+import org.I0Itec.zkclient.exception.ZkException;
+import org.apache.zookeeper.KeeperException;
+
+
+/**
+ * Exception thrown when an action is taken by an expected zk session which
+ * does not match the actual zk session.
+ */
+public class ZkSessionMismatchedException extends ZkException {
+
+    private static final long serialVersionUID = 1L;
+
+    public ZkSessionMismatchedException() {
+        super();
+    }
+
+    public ZkSessionMismatchedException(KeeperException cause) {
```

Comment:
I don't think we will have a KeeperException to cause ZkSessionMismatchedException. Since ZkSessionMismatchedException is basically invented by us in this PR. So this construction might be unnecessary.